palantirnet/federated-search-react
# UMUS-89: Default theme and custom theming support

This PR creates a default theme for the search app and enables custom theming.

## JIRA
[UMUS-89](https://palantir.atlassian.net/browse/UMUS-89)

## Related
- Custom theme implementation on health and lab blog sites: palantirnet/umhs-acquia#16
- Update module to implement default search theme and document theming workflow palantirnet/search_api_federated_solr#20

## To Test
1. Checkout this branch
1. Run `yarn install` to install project dependencies
1. Copy `./src/.env.js.example` into `./src/.env.js` (if it's not still there)
1. Add https://ss826806-us-east-1-aws.measuredsearch.com:443/solr/master/select as the url
1. Add any other values to the available options
1. Run `yarn start` to load the dev environment
1. Observe the new `./theme/search_theme_override.scss` file
1. Uncomment the code in the `./theme/search_theme_override.scss` file and save
1. Observe the search app reloads with new styles
1. Quit the dev workflow `CTRL + C`
1. Build the production assets to confirm that your custom styles are *not* added by running `yarn run build`
1. Your `git status` should indicate no changes to the build assets
